UnityProject/Assets/Scripts/Items/Food/Torch.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using Mirror;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Random = UnityEngine.Random;
+using Items;
+
+namespace items
+{
+    /// <summary>
+    /// Lightable objects that can be used as lighters
+    /// </summary>
+    public class Torch : NetworkBehaviour, ICheckedInteractable<HandApply>,
```

Comment:
why inside the ``Food`` folder?

Reply:
Because that is the location of both lighters and cigarettes

Reply:
fair